web-registry/src/components/organisms/AppFooter.tsx

```diff
@@ -6,71 +6,95 @@ import {
   FooterItemProps as FooterItem,
 } from 'web-components/lib/components/footer/footer-new';
 
-import { chainId, nctBasket } from '../../lib/ledger';
 import { Link, RegistryIconLink } from '../atoms';
 
 const AppFooter: React.FC = () => {
   const { pathname } = useLocation();
   const isHidden = ['/project-pages'].some(route => pathname.startsWith(route));
 
-  const footerItems: [FooterItem, FooterItem, FooterItem] = [
+  const footerItems: [FooterItem, FooterItem, FooterItem, FooterItem] = [
     {
-      title: 'Registry App',
+      title: 'Explore',
       items: [
         {
           title: 'Projects',
-          href: '/#projects',
+          href: '/projects',
         },
+        {
+          title: 'Credit Classes',
+          href: '/',
+        },
+        // {
+        //   title: 'Baskets',
+        //   href: '/',
+        // },
```

Comment:
Should we remove all these commented items?
We can always check history if we need to get them back.

Reply:
lgtm!